tests/xtradbcluster/xtra_db_cluster_test.go

```diff
@@ -84,14 +94,60 @@ func TestXtraDBClusterAPI(t *testing.T) {
 
 	for _, cluster := range clusters.Clusters {
 		if cluster.Name == name {
-			assert.Equal(t, int32(3), cluster.Params.ClusterSize)
+			assert.Equal(t, int32(1), cluster.Params.ClusterSize)
 			assert.Equal(t, int64(1024*1024*1024), cluster.Params.Proxysql.ComputeResources.MemoryBytes)
 			assert.Equal(t, int32(1000), cluster.Params.Proxysql.ComputeResources.CpuM)
 			clusterFound = true
 		}
 	}
 	assert.True(t, clusterFound)
 
+	updateClusterReq := &controllerv1beta1.UpdateXtraDBClusterRequest{
```

Comment:
the same comments for pxc too

Reply:
Done with the same limitations. 
Only updated memory due to resources limitation.